stattlepy/Stattleship_API.py

```diff
@@ -42,22 +60,57 @@ def ss_get_results(self,**kwargs):
                         elif str(key) == 'verbose':
                                 verbose = value
                         elif str(key) == 'stat_type':
-                                param['type'] = value  
+                             param['type'] = value
                         else:
                                 param[key] = value
                 
-                
-                if(verbose):
-                    print 'Making Initial API Request'
+                ### initial verbose to indicate request occurring
+                if verbose:
+                    print'Making Initial API Request'
                     
-                ###NOTE WALK IS NOT YET SUPPORTED, IN DEV
-                        
-                tmp = self.query_api(sport, league, ep, param, version, walk, page, verbose, token )
+                ### initial query       
+                tmp, return_header = self.query_api(sport, league, ep, param, version, walk, page, verbose, token)
                
+                ### make response list
                 response = list()
                
-                response.append(tmp)       
-               
+                ### set the original first parsed 
+                response.append(tmp)
+                
+                ### walk function using REGEX to idenify the next link in the header to pull next request 
+                if(walk):
+                    while 'link' in return_header:
+                        
+                        ### Next link to request from the API
+                        next_link = re.findall( 'rel="last", <(.*?)>; rel="next"', return_header['link'], re.MULTILINE)
+                        
+                        ### Use try and except to see if the next link was found within the header
+                        try:
+                            if verbose:
+                                print 'Next link sent to API:'
+                                print next_link[0]
+
+                            headers = {
+                            'Authorization': token,
+                            'Accept':'application/vnd.stattleship.com; version=%s' %version,
+                            'Content-Type':'application/json'        
```

Comment:
@AdamMJenks We've started to measure analytics on each UserAgent so we know which libraries are using the API.

Could you:

1) Version this package (0.0.1)
2) Include a User-Agent header in the request:

```
        'User-Agent':
          "Stattleship Python/#{your version} (#{PLATFORM})"
```

As in

`Stattleship R/0.0.4 (x86_64-w64-mingw32/x64 (64-bit))`

Where `PLATFORM` is `(x86_64-w64-mingw32/x64 (64-bit))`